rocrate_validator/profiles/five-safes-crate/must/4_sign_off.ttl

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2025 eScience Lab, The University of Manchester
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+@prefix ro: <./> .
+@prefix ro-crate: <https://github.com/crs4/rocrate-validator/profiles/ro-crate/> .
+@prefix five-safes-crate: <https://github.com/eScienceLab/rocrate-validator/profiles/five-safes-crate/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix schema: <http://schema.org/> .
+@prefix purl: <http://purl.org/dc/terms/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix validator: <https://github.com/crs4/rocrate-validator/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+five-safes-crate:SignOffObjectActionAndName
+    a sh:NodeShape ;
+    sh:name "SignOff" ;
+    sh:description "Sign Off phase" ;
+
+    sh:target [
+        a sh:SPARQLTarget ;
+        sh:select """
+            PREFIX schema: <http://schema.org/>
+            PREFIX shp:    <https://w3id.org/shp#>
+            SELECT ?this
+            WHERE {
+                ?this schema:additionalType shp:SignOff .
+            }
+        """ ;
+    ] ;
+
+    sh:property [
+        sh:path schema:name ;
+        sh:datatype xsd:string ;
+        sh:minCount 1 ;
+        sh:severity sh:Violation ;
+        sh:message "Sign Off phase MUST have a human-readable name string." ;
+    ] ;
+
+    sh:property [
+        sh:path rdf:type ;
+        sh:minCount 1 ;
+        sh:hasValue schema:AssessAction;
+        sh:severity sh:Violation ;
+        sh:message "Sign Off phase MUST be a `schema:AssessAction`." ;
```

Comment:
Just noting after wrangling a more complex type check https://github.com/eScienceLab/rocrate-validator/pull/36#discussion_r2588100441 that using `sh:class` on the NodeShape (not within the property) is a more built-in way to achieve this ([docs](https://book.validatingrdf.com/bookHtml011.html#sec131)) - BUT maybe doesn't offer the necessary granularity of messaging and severity when combined with other checks in the same NodeShape.

If this works as it is, it's fine to keep.